### DISCUSSION TITLE
关于网页安装ipa，安装成功后，图标打不开

### DISCUSSION THREAD

**Discussion by alinjx** [AlistGo/alist#775] [General]
自签后，爱思助手可以正常安装使用；但是使用alist的在线安装，桌面出现图标并且转圈完成，但是打开的时候，提示安装失败。有没有大神懂，希望可以指点一下这是为什么。

**Comment by zhlx2835** [comment]
这个 肯定是 需要配置的，具体咋配置我也不会，也在求教程方法

**Comment by anwen-anyi** [comment]
=，= 是需要签名的..
